src/authorizations/components/GenerateTokenDropdown.tsx

```diff
@@ -30,9 +30,15 @@ const GenerateTokenDropdown: FC<ReduxProps & GenerateTokenProps> = ({
 
   const customApiOption = 'Custom API Token'
 
-  const handleAllAccess = () => {
-    showOverlay('add-master-token', null, dismissOverlay)
-    event('generate_token_dropdown.all_access_overlay.opened')
+  const handleAllAccess = async () => {
+    try {
+      await getAllResources()
+      showOverlay('add-master-token', null, dismissOverlay)
+      event('generate_token_dropdown.all_access_overlay.opened')
+    } catch (e) {
+      dispatch(notify(getResourcesTokensFailure()))
```

Comment:
ohh just realized this; in the event `getAllResources` fails to fetch the data and `getResourcesTokensFailure` notification is fired, the message that pops up  will read `Failed to fetch all resources for creating custom api token`. I think the message should say something like Failed to fetch all resources for creating an _all access_ api token.

Reply:
![Screen Shot 2022-02-23 at 5 43 30 PM](https://user-images.githubusercontent.com/66275100/155429479-3bdd04a9-f0cf-4696-99fc-850c5c7617e2.png)
maybe we should create another function that handles failure notification specifically for all access tokens?

Reply:
Good catch! Yes, we need the correct message for the All Access Token.